asset/css/site-nav.css

```diff
@@ -30,14 +30,14 @@
     -webkit-transition: 0.4s;
 }
 
-#site-nav a:active,
-#site-nav a:link,
-#site-nav a:visited {
+.site-nav__a:link,
+.site-nav__a:visited,
+.site-nav__a:active {
```

Comment:
Should be alphabetical order actually? :P

Reply:
> a:hover MUST come after a:link and a:visited
> a:active MUST come after a:hover
> [Source](https://www.w3schools.com/css/css_link.asp)

So active being the last is correct.

Reply:
That's for the case where the link states have different styles.
I'm okay with that order here, but let's add that reference as a comment.

Reply:
I have added a link to the relevant section in the CSS standard for a more authoritative reference!